### DISCUSSION TITLE
IPFS Desktop on Windows not loading

### DISCUSSION THREAD

**Discussion by Ikpong-Joseph** [Cyfrin/foundry-full-course-cu#1550] [Q&A]
Hey guys, I tried setting up my IPFS Desktop on a Windows PC and I keep getting this error page.
![Screenshot (69)](https://github.com/Cyfrin/foundry-full-course-f23/assets/150890288/10d81687-20e1-4225-a526-9e44b4647e40)

I've tried adding Kubo, but still get an error.
![Screenshot (70)](https://github.com/Cyfrin/foundry-full-course-f23/assets/150890288/54adfd67-f9e3-4638-a81e-d77d48f0b5dc)

Any tips? Gracias.

**Comment by EngrPips** [comment]
Hello @Ikpong-Joseph , first of all what is `Kubo` and i have never run into these error before but it appears to me that your internet connection might be poor.

**Reply by Ikpong-Joseph** [reply]
Thought so for a minute. Connected to a VPN, but same error. Uninstalled and reinstalled and the error didn't show up on the first launch. Subsequent launches erred.

**Reply by EngrPips** [reply]
That seem really weird i would say
